src/service.js

```diff
@@ -145,7 +146,7 @@ class Service {
 						"_start",
 						"_stop",
 						"_init"
-					].indexOf(name) != -1
+					].indexOf(name) !== -1
 				) {
 					throw new ServiceSchemaError(
```

Comment:
I've just known why these methods was static because they are not conflict with user-defined methods. 
To avoid it, please add a new check here and throw the same error, if the name  starts with `mergeSchema` or equals `applyMixins`